packages/vanilla/src/components/select/select.stories.js

```diff
@@ -102,5 +101,6 @@ Select.args = {
       label: 'Port of Zoolily'
     },
   },
-  disabled: false
+  disabled: false,
```

Comment:
Why not leave these unset? That should be the default state, IMO. Just don't pass them as args unless you need to set it. Not going to hold this up over this one, so your call if you update or not.